includes/class-wc-payments-account.php

```diff
@@ -476,6 +476,12 @@ public function maybe_redirect_to_onboarding() {
 			return false;
 		}
 
+		// Redirect directly to onboarding page if come from WC Admin task and are in treatment mode.
+		$http_referer = sanitize_text_field( wp_unslash( $_SERVER['HTTP_REFERER'] ?? '' ) );
```

Comment:
I think it would be good to cover this if in the unit test as well!

Reply:
🤔  I can't think of an easy way to test it, because `is_in_onboarding_treatment_mode` is a static method, and we can't mock it, right?

We made it static as all the other utils methods, to easily call it from different places. Probably could be a better option to make `Experimental_Abtest` a singleton. But AFAIK we're using it as a temporal replacement while it's being included in Jetpack.

Reply:
Yeah, you're right, it would require additional moves.
p.s. I did not know that `Experimental_Abtest` is temporary!